src/components/Expression/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Root } from "./Root";
+import { OperandDropdown, OperantDropdownItem } from "./OperandDropdown";
+import { AutocompleteItem, OperandAutocomplete } from "./OperandAutocomplete";
+import { Condition, ConditionItem } from "./Condition";
+import { OperandText } from "./OperandText";
+import { OperandNumber } from "./OperandNumber";
+import { OperandRange } from "./OperandRange";
+
```

Comment:
**suggestion**: you need to export types here as well e.g from `Expression/Root.tsx`

Reply:
This can be added later as well - it is not blocking merge 😄